english/alchemy/alchemy.md

```diff
@@ -24,7 +24,7 @@ forwards - studying its imperfection - a ring like shape.
 Looking through the hole, the air shimmers, giving the other side a
 hazy appearance.
 The objects on the other side appear to be vacillating, as if they were
-breathing.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.